lib/components/BuilderUI.jsx

```diff
@@ -10,7 +11,7 @@ import { TxMessage } from 'lib/components/TxMessage'
 import { WalletContext } from 'lib/components/WalletContextProvider'
 import { poolToast } from 'lib/utils/poolToast'
 
-
+const now = () => Math.floor((new Date()).getTime() / 1000)
```

Comment:
Out of curiousity's sake why are you rounding this down?

Reply:
Rounding down yields a time in the past, whereas rounding up could yield a time sightly in the future.  Guess it doesn't matter here, just feels more natural